app/assets/javascripts/moss.js

```diff
@@ -0,0 +1,32 @@
+function toggleOptions(dropdown, table) {
```

Comment:
It might be nice to move the component code to a separate js file for portability

Reply:
Moved to `dropdown.js`